### DISCUSSION TITLE
Easy Send to

### DISCUSSION THREAD

**Discussion by Liquidey** [localsend/localsend#1352] [Discussion (English, International)] [answered]
I'm asking for a feature that adds the option to share via the Windows context menu
send to
**Like this** 
![image](https://github.com/localsend/localsend/assets/77444420/78495a15-b2b8-4654-a493-873da0d290ac)
Sorry if i misspell
Btw is my first time on github sorry if not writing in right place

**Comment by samocvetov** [comment]
https://www.sordum.org/10830/sendto-menu-editor-v1-3/

**Reply by Otus159** [reply, marked as answer]
This will be a good feature also
